### PR TITLE
fix(mongo): mongo datasource throws an error during introspection with mongoose 6 as dependency

### PR DESCRIPTION
**Pull request by Scra3**
linked to: CU-86bzfa0d2
### General

- [ ] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org)
- [ ] Test manually the implemented changes
- [ ] Validate the code quality (indentation, syntax, style, simplicity, readability)

### Security

- [ ] Consider the security impact of the changes made

### CODE REVIEW

File: packages/datasource-mongo/package.json [hunk 1]
```diff
@@ -15,9 +15,11 @@
     "@forestadmin/datasource-mongoose": "1.7.7",
     "@forestadmin/datasource-toolkit": "1.34.0",
     "json-stringify-pretty-compact": "^3.0.0",
-    "mongoose": "^7.0.1",
     "tunnel-ssh": "^5.1.2"
   },
+  "devDependencies": {
+    "mongoose": "^7.0.1"
+  },
```
Comment (Thenkei) [inline packages/datasource-mongo/package.json:22]: We will need to be careful with cloud ! And also for our client that don't installed `mongoose` when using mongo DS.

Comment (Thenkei) [reply packages/datasource-mongo/package.json:22]: Well in DS SQL 👀 

https://github.com/ForestAdmin/agent-nodejs/blob/16b14a50c74e418d9caa83345e883660a576d176/packages/datasource-sql/package.json#L18

Comment (Scra3) [reply packages/datasource-mongo/package.json]: Fixed. I removed it and tell to our customer to use resolution or override.

**Review by Thenkei — APPROVED** [review]
Works for me. :)

**Pull request merged**